src/lib/sub-app/sub-app.factory.ts

```diff
@@ -252,26 +253,29 @@ function moveDefaultAppToApps(
     if (process.env.NODE_ENV === TEST_ENV) {
       return host;
     }
-    try {
-      if (fse.existsSync(sourceRoot)) {
-        fse.moveSync(
-          sourceRoot,
-          join(projectRoot as Path, appName, sourceRoot),
-        );
-      }
-      const testDir = 'test';
-      if (fse.existsSync(testDir)) {
-        fse.moveSync(testDir, join(projectRoot as Path, appName, testDir));
-      }
-    } catch (err) {
-      throw new SchematicsException(
-        `The "${projectRoot}" directory exists already.`,
-      );
-    }
+    // mv src
+    moveDirectories(sourceRoot, join(projectRoot as Path, appName), host);
+
+    // mv tests
+    const testDir = 'test';
+    moveDirectories(testDir, join(projectRoot as Path, appName), host);
```

Comment:
```suggestion
    moveDirectories(sourceRoot, join(projectRoot as Path, appName), host);

    const testDir = 'test';
    moveDirectories(testDir, join(projectRoot as Path, appName), host);
```
code seems to be self-explanatory(?)

Reply:
I've removed these comments :heavy_check_mark: